exporters/cr/exporter.go

```diff
@@ -56,7 +76,9 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 	for _, namespace := range namespacesToCheck {
 		certs, err := e.dynamicClient.Resource(certManagerCertificateGroupVersionResource).Namespace(namespace).List(e.ctx, listOptions)
 		if err != nil {
-			e.logger.Log("error", microerror.Mask(err))
+			if !errors.IsNotFound(err) {
+				e.logger.Log("error", microerror.Mask(err))
+			}
```

Comment:
@yulianedyalkova 

Is this okay for you?

Related: https://github.com/giantswarm/cert-exporter/pull/169#discussion_r704289502

Reply:
For my interest. So because we use the dynamic client we get an IsNotFound error?

If so I think that's fine. 
Its also nicer than generated clients where you need to handle the CRD not being present.

Reply:
Yeah I am using dynamic client because I didn't want to include cert-manager dependency.

Reply:
Nice, thanks for confirming and better to keep the dependencies clean.